code-studio/src/js/components/Grid.jsx

```diff
@@ -56,7 +58,13 @@ var Cell = React.createClass({
       }
     }
 
-    return (<td className={classNames.join(' ')} onClick={this.handleClickCell} style={tdStyle}>
+    return (<td
+        className={classNames.join(' ')}
```

Comment:
Nit: [Styleguide suggests](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#react) your wrapping parentheses should be on different lines than your open/close tags, to ensure the tags line up.
```jsx
return (
  <td
      className={classNames.join(' ')}
      ...>
    {text}
  </td>
);
```